fjord/settings/base.py

```diff
@@ -9,13 +9,8 @@
 
 from fjord.settings_utils import config
 
-
-ROOT = os.path.abspath(
-    os.path.join(
-        os.path.dirname(__file__),
-        '..',
-        '..'
-    ))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Django 1.6 introduced `BASE_DIR`. I added it, but we only use it on the next line. I figured it might make future things easier since it's a standard setting in the Django template.